vulcanai2/models/cnn.py

```diff
@@ -161,21 +159,50 @@ def _build_conv_network(self, conv_hid_layers, activation):
             the conv network as a nn.Sequential object
 
         """
+        conv_hid_layers[0]['in_channels'] = self._in_dim[0][0]
         conv_layers = []
         for conv_layer_config in conv_hid_layers:
             conv_layer_config['activation'] = activation
             conv_layers.append(ConvUnit(**conv_layer_config))
         conv_network = nn.Sequential(*conv_layers)
         return conv_network
-    
+
+    def _create_classification_layer(self, dim, pred_activation):
+        self.network_tail = DenseUnit(
+            in_features=dim,
+            out_features=self.out_dim,
+            activation=pred_activation)
+
+    def _forward(self, xs, **kwargs):
+        """
+        Computation for the forward pass of the ConvNet module.
+
+        Parameters
+        ----------
+        xs : list(torch.Tensor)
+            List of input tensors to pass through self.
+
+        Returns
+        -------
+        output : torch.Tensor
+
+        """
+        out = []
+        for x in xs:
```

Comment:
What is the point of this? you're not running it through any network. might as well just cat `xs` no?

Reply:
yeah we are using `torch.cat` in the end. the input tensors needed to be formatted before passing into the network, it will be implemented soon.